### DISCUSSION TITLE
Error in check lvs design sky130hd/gcd

### DISCUSSION THREAD

**Discussion by hoanhe** [The-OpenROAD-Project/OpenROAD#3379] [Q&A]
Hello
When I check lvs for sky130hd/gcd, I have an error about cdl file. 

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/134496656/49ad14ce-fa0c-408e-b0b4-37fa17d5a6d5)

Please help me to solve this problem!

**Comment by rovinski** [comment]
Please file an issue with a reproducible test case.

**Reply by vijayank88** [reply]
`make lvs` is work in progress. Not yet implemented fully. Only `make drc` implemented for physical verification stage. @hoanhe 
If you had development in `make lvs` to complete it, you can contribute it.
